lighthouse-core/gather/gatherers/dobetterweb/optimized-images.js

```diff
@@ -129,8 +129,6 @@ class OptimizedImages extends Gatherer {
    * @return {Promise<?{fromProtocol: boolean, originalSize: number, jpegSize: number, webpSize: number}>}
    */
   calculateImageStats(driver, networkRecord) {
-    // TODO(phulce): remove this dance of trying _getEncodedResponse with a fallback when Audits
```

Comment:
at this point, I'm not sure it makes sense. It's a solid fallback option for when the protocol fails.